### DISCUSSION TITLE
Where are the data in pv stored in ceph?

### DISCUSSION THREAD

**Discussion by chominnn** [rook/rook#14654] [Q&A]
if i create persistent volume with rook-ceph (storage class) Where are the data in pv stored in ceph?

I found that PV is stored in CSI-vol form

but cant open csi-vol

I'd like to have multiple pods store files in one directory (?) within the ceph.

**Comment by Madhu-1** [comment]
@chominnn if its a cephfs pvc the data will be stored in the subvolume and for rbd its stored in the rbd image, you can use https://rook.io/docs/rook/latest-release/Troubleshooting/direct-tools/ this to access it.

**Reply by chominnn** [reply]
thx, i will try today

**Comment by travisn** [comment]
@chominnn See the [Filesystem topic](https://rook.io/docs/rook/latest/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage/) for creating shared volumes.
